src/scss/misc.scss

```diff
@@ -391,4 +391,12 @@ app-root > #loading {
             }
         }
     }
+
+    &[role="alert"] {
```

Comment:
Does it need to be limited to role=alert? Should this be for all callouts?

Reply:
Perhaps, not sure what else uses `.callout` with nested `<ul>`; I'll let Vince give it a quick look

Reply:
This is only example for the `desktop` application that uses a `.callout` with nested `<ul>`

Reply:
But all callouts will have `&[role="alert"]`, which is why i think this is unncessary. Just style all `ul` that are nested in a `.callout`

Reply:
I'm on board with that, thanks Kyle

Reply:
Followed suggestions above. Updated in latest push and is working as intended.